### ISSUE TITLE
mixins/variables from imports unavailable in other imports

### ISSUE THREAD

**Issue by jkpubsrc** [less/less.js#3580]
I'm not perfectly clear if this here is the right place to open such an issue as my issue targets a general design aspect of the less language not with `less.js` in particular. As `include` does not work with mixins I discovered this issue https://github.com/less/less.ruby/issues/65 at `less.ruby` which is exactly the issue I have with `less.js`. (But as far as I understand it is of general nature and not limited to `less.ruby` or `less.js`.) I open this issue here for organizational purposes as a) I think it *should* be addressed quite soon and b) I personally encountered this issue regarding `less.js` (not: `less.ruby`) and c) I'm not sure where to put it otherwise where it would be more appropriate.

In short: If `include` is used to include another `less` file, the mixins are not imported. This is suboptimal as this prevents me from modularizing my `less` file and giving it a better structure.

More details: https://github.com/less/less.ruby/issues/65

**Comment by matthew-dean** [comment]
You're referencing a very old issue. You can import mixins / vars from other files.

**Comment by jkpubsrc** [comment]
Please define "very old": I've use the release available online at dec 23rd. And I just checked:

```
$ pip3 install lesscpy -U
Requirement already satisfied: lesscpy in /usr/local/lib/python3.8/dist-packages (0.14.0)
Requirement already satisfied: ply in /usr/local/lib/python3.8/dist-packages (from lesscpy) (3.11)
Requirement already satisfied: six in /usr/lib/python3/dist-packages (from lesscpy) (1.14.0)
```

So I'm using the most recent release provided.
